websites/P/Paramount+/presence.ts

```diff
@@ -1,153 +1,186 @@
+				);
+
+				video = document.querySelector("video");
+
+				title = data.partOfSeries.name;
+
+				seasonEpisode = `S${data.partOfSeason.seasonNumber}:E${data.episodeNumber} - ${data.name}`;
+
+				if (title) presenceData.details = title;
+
+				if (seasonEpisode) presenceData.state = seasonEpisode;
+
+				presenceData.smallImageKey = video.paused ? Assets.Pause : Assets.Play;
+				presenceData.smallImageText = video.paused
+					? strings.pause
+					: strings.play;
+				presenceData.largeImageKey = data.image || Logos.Paramount;
+
+				[presenceData.startTimestamp, presenceData.endTimestamp] =
+					presence.getTimestampsfromMedia(video);
+
+				if (video.paused) {
+					delete presenceData.startTimestamp;
+					delete presenceData.endTimestamp;
+				}
+			} else if (
+				!vidArea &&
+				document.querySelector('[type="application/ld+json"]') !== null
+			) {
+				presenceData.details = "Viewing series";
+				presenceData.state = JSON.parse(
+					document.querySelector('[type="application/ld+json"]').textContent
+				).name;
+			} else {
+				presenceData.details = "Browsing";
+				presenceData.state = "Viewing Shows";
 			}
+			break;
+		}
 
-			const content = seasonEpi;
-
-			if (movTitle) {
-				presenceData.state = movTitle;
-				presenceData.details = "Watching Movie:";
+		case pathIncludes(path, "/movies"):
+			if (
+				vidArea &&
+				document.querySelector("video.marqueeVideo") &&
+				!vidArea.querySelector("video")
+			) {
+				presenceData.details = "Previewing a movie";
+				presenceData.state = JSON.parse(
+					document.querySelector('[type="application/ld+json"]').textContent
+				).name;
+			} else if (vidArea && vidArea.querySelector("video")) {
+				const movData = JSON.parse(
+					document.querySelector('[type="application/ld+json"]').textContent
+				);
+
+				video = vidArea.querySelector("video");
+
+				if (movData.name) presenceData.state = movData.name;
+				presenceData.details = "Watching a movie";
+
+				presenceData.smallImageKey = video.paused ? Assets.Pause : Assets.Play;
+				presenceData.smallImageText = video.paused
+					? strings.pause
+					: strings.play;
+
+				[presenceData.startTimestamp, presenceData.endTimestamp] =
+					presence.getTimestampsfromMedia(video);
+
+				if (video.paused) {
+					delete presenceData.startTimestamp;
+					delete presenceData.endTimestamp;
+				}
+			} else {
+				presenceData.details = "Browsing";
+				presenceData.state = "Viewing Movies";
+			}
+			break;
+
+		case pathIncludes(path, "/live"):
+			liveTitle = document.querySelector(
+				".video__metadata.padded-container > p"
+			).textContent;
+
+			presenceData.details = "Watching Live TV";
+			presenceData.state = liveTitle;
+			presenceData.smallImageKey = Assets.Live;
+			presenceData.smallImageText = strings.live;
+			break;
+
+		case pathIncludes(path, "/brands"):
+			if (path.includes("/cbs/")) {
+				presenceData.details = "Browsing Brand:";
```

Comment:
Prefer strings without `:`. Also, this line is duplicated many times in the if/else branches. It may be better to just put it before the first if statement.
```ts
presenceData.details = "Browsing Brand";
```